generator/test/Test/Pos/Block/LrcSpec.hs

```diff
@@ -1,10 +1,10 @@
 {-# LANGUAGE TypeFamilies #-}
 
--- | Specification of 'Pos.Lrc.Worker' (actually only
+-- | Specification of 'Pos.Block.Lrc' (actually only
 -- 'lrcSingleShotNoLock' which probably shouldn't be there, but it
 -- doesn't matter now).
 
-module Test.Pos.Lrc.WorkerSpec
+module Test.Pos.Block.LrcSpec
```

Comment:
If this module is in the `generator` package then its name should be `Test.Pos.Generator.Block.LrcSpec` I think.